microprofile/testing/junit5/src/main/java/io/helidon/microprofile/testing/junit5/HelidonJunitExtension.java

```diff
@@ -318,16 +315,17 @@ private void configure(ConfigMeta configMeta) {
             ConfigBuilder builder = configProviderResolver.getBuilder();
 
             configMeta.additionalSources.forEach(it -> {
-                // If not using a YAML extension, assume properties file
                 String fileName = it.trim();
-                if (YAML_SUFFIXES.stream().anyMatch(fileName::endsWith)) {
-                    builder.withSources(YamlMpConfigSource.classPath(it).toArray(new ConfigSource[0]));
-                } else {
-                    builder.withSources(MpConfigSources.classPath(it).toArray(new ConfigSource[0]));
+                String type = fileName.substring(fileName.lastIndexOf('.') + 1);
```

Comment:
This could will fail with `ArrayIndexOutOfBounds` for files that do not have a suffix. That should not happen - i.e. `my-conifguration` file would fail.
The original code always falls back to `properties`, please fix this to behave the same.

Reply:
It will be a user fault, but OK, I will update it.